econml/drlearner.py

```diff
@@ -1,1188 +1,1188 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
The entire file is showing up as modified on GitHub; when I do a diff locally and set it to ignore whitespace I see the expected changes.  Is it possible that the line endings have changed here?  Normally git automatically handles this, but please take a look and fix on your end.

Reply:
Weird! What can I do?

Reply:
Looks like you could try `git add --renormalize` on the file, assuming your version of git is relatively recent.

Reply:
fixed!